import 'package:checked_exceptions_annotations/checked_exceptions_annotations.dart';

@safe
void foo() {
  throw Exception();
}
